Woodoku/tests/Entity/test_woodoku_board.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
```

Comment:
file in the wrong folder

Reply:
What do you mean?

Reply:
**
![image](https://user-images.githubusercontent.com/42769475/180589853-2627e6a3-0136-448d-895d-0f3af8fb2c45.png)
**
file in the wrong folder